packages/teams-js/src/public/authentication.ts

```diff
@@ -395,54 +327,6 @@ export namespace authentication {
     removeHandler('navigateCrossDomain');
   }
```

Comment:
If we are deleting the start code presumably we can also delete the stop code?

Reply:
We cannot delete this code as it's used in `initialize` function indirectly. When teamsJS is `initialized()` we register two handlers `authentication.authenticate.success` and `authentication.authenticate.failure` with `handleSuccess` and `handleFailure` respectively. Even if the auth window is deferred to the host, we need to monitor that the popped up window is closed and what was the message returned(success/failure). This is my understanding, if you need more explanation, I'm happy to talk.

Reply:
I tried it out and apparently, we don't need this code anymore as it'll be handled by hosts.